internal/namespaces/instance/v1/custom_user_data_test.go

```diff
@@ -7,15 +7,6 @@ import (
 )
 
 func Test_UserDataGet(t *testing.T) {
-	t.Run("Usage", core.Test(&core.TestConfig{
```

Comment:
There must be some goldens to delete for all the removed test

Reply:
I deleted a few goldens, but they were recreated by the new test. 

It looks like those files were moved.